src/server.js

```diff
@@ -34,7 +34,7 @@ export default ({onError, CsrfProtection}) => {
       const script = html`
 <script nonce="${ctx.nonce}">
 onerror = function(m,s,l,c,e) {
-  if (e.__handled) return;
+  if (!e || e.__handled) return;
```

Comment:
should we not create a new Error() and fill in all the rest of the details`m,s,l,c` as received ?